pinc/Quiz.inc

```diff
@@ -168,28 +168,30 @@ class Quiz
             {
                 $passed = user_has_passed_quiz_page($username, $quiz_page_id);
                 $text = $passed ? _("Passed") : _("Not passed");
-                $bgcolor = $passed ? '#ccffcc' : '#ffcccc';
-                echo "<td style='background-color: $bgcolor'>$text</td>";
+                $bgcolor = $passed ? 'pass' : 'no-pass';
```

Comment:
Rename `$bgcolor` to `$class` or someting similar since it no longer contains a color (here and below).

Reply:
Also consider prefixing the class names with `quiz-` so `quiz-pass` & `quiz-no-pass` (or `quiz-not-passed`?).

Reply:
Done. Coming up with meaningful, relatively short class names can be a challenge, but you're right that it's better to be more detailed. I was also trying to keep from giving the color variables and the classes the same names, but ...